### DISCUSSION TITLE
How do I load data from HDFS with separator "\001"(Hive default separator)

### DISCUSSION THREAD

**Discussion by tinkerrrr** [apache/doris#6780] [A - General / Q&A]
I simply used `COLUMNS TERMINATED BY "\001"`, but got error:

```
Reason: actual column number is less than schema column number. actual number: 4 sep: \, schema number: 29;
```

**Comment by EmmyMiao87** [comment]
What's your Doris version? 
We support multiple column sep recently, maybe 0.14 can support

**Reply by tinkerrrr** [reply]
The version is 0.14, but how to use multiple column separator?

**Reply by EmmyMiao87** [reply]
This is the PR of supporting multi separator : https://github.com/apache/incubator-doris/pull/5462
You can check whether your version include it
